src/xrpld/app/ledger/OrderBookDB.cpp

```diff
@@ -129,8 +129,8 @@ OrderBookDB::update(std::shared_ptr<ReadView const> const& ledger)
             }
             else if (sle->getType() == ltAMM)
             {
-                auto const issue1 = (*sle)[sfAsset];
-                auto const issue2 = (*sle)[sfAsset2];
+                auto const issue1 = (*sle)[sfAsset].get<Issue>();
+                auto const issue2 = (*sle)[sfAsset2].get<Issue>();
```

Comment:
I just noticed that unit tests never execute this block, and they did not execute it before your change either.

Would be great if you can add it to unit tests.

Reply:
I'd recommend postponing this. It has nothing to do with `STIssue` PR and it was never executed. Perhaps could consider it as part of PR #5203.